packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/components/policy-form/controller.js

```diff
@@ -2,13 +2,23 @@ import { ACTIONS } from '../../constants';
 import { ACTIONS_LIST, URL_PATTERN, URL_PLACEHOLDER } from './constants';
 
 export default class OctaviaLoadBalancerL7PolicyFormCtrl {
-  constructor() {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
     this.ACTIONS = ACTIONS;
     this.URL_PATTERN = URL_PATTERN;
     this.URL_PLACEHOLDER = URL_PLACEHOLDER;
     this.actions = ACTIONS_LIST;
   }
 
+  $onInit() {
+    this.submitLabel = this.submitLabel
+      ? this.submitLabel
+      : this.$translate.instant(
+          'octavia_load_balancer_create_l7_policy_submit',
+        );
```

Comment:
Sounds more logic to store the value in a new property when it's possible to override a default value.
Moreover, the given value `submitLabel` has to be translated as well. 
```suggestion
    this.submitBtnLabel = this.$translate.instant(
           this.submitLabel ?? 'octavia_load_balancer_create_l7_policy_submit',
        );
```